package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "private": true
```

Comment:
Nothing! This was something I had seen due to not having a published (node) project. But after testing the build procedure, we can just have an empty JSON object and it be more than fine! I will fix that.